internal/app/bootkube/assets.go

```diff
@@ -0,0 +1,311 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
contents of this file were just literally moved from the service file, no changes